main.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // Version holds the current Gitea version
-var Version = "1.5.0-dev"
+var Version = "1.8.0-dev-111"
```

Comment:
This should be set to "0", as makefile overrides it.

Reply:
I changed this to 1.9.0-dev, it should be better than 1.5.0-dev. I think there is another PR that makefile overrides it.